tap_zendesk/tap.py

```diff
@@ -35,10 +35,9 @@ class TapZendesk(Tap):
             description="The earliest record date to sync",
         ),
         th.Property(
-            "api_url",
-            th.StringType,
-            default="https://api.mysample.com",
```

Comment:
I'm missing here the other params defined in meltano:

```

    - name: email
      label: Email
      description: This is the email you use to login to your Zendesk dashboard. For
        API Authentication, `/token` is automatically appended to the email address
        and is not required in the configuration.
    - name: api_token
      label: API Token
      description: See 
        https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-
        for more details about generating an API Token.
    - name: access_token
      label: Access Token
      description: To use OAuth, you will need to fetch an `access_token` from a configured
        Zendesk integration.
    - name: subdomain
      label: Zendesk Subdomain
      description: The subdomain of your Zendesk instance.
    - name: start_date
      kind: date_iso8601
      label: Start Date
      description: Determines how much historical data will be extracted. Please be
        aware that the larger the time period and amount of data, the longer the initial
        extraction can be expected to take.
    - name: end_date
      kind: date_iso8601
      label: End Date
      description: Sets a limit on how much data to extract in combination with start_date.
        If passed, data would be loaded for date >= start_date and date < end_date.
    - name: min_remain_rate_limit
      kind: integer
      label: Minimum Remain Rate Limit
      description: Sets a limit to the remain rate that the tap will not will not
        overtake (it will wait for the reset)
    - name: ticket_paginate_by_time
      kind: boolean
      label: Ticket pagination by Time
      description: (Default True) Sets ticket pagination by time-based (True) or cursor-based
        (False).
```

Reply:
Fixed! I removed `ticket_paginate_by_time` as we don't use it.